modules/openthread/platform/crypto_psa.c

```diff
@@ -287,6 +295,160 @@ otError otPlatCryptoHmacSha256Finish(otCryptoContext *aContext, uint8_t *aBuf, s
 	return psaToOtError(psa_mac_sign_finish(operation, aBuf, aBufLength, &mac_length));
 }
 
+
+otError otPlatCryptoHkdfInit(otCryptoContext *aContext)
+{
+	otError error = OT_ERROR_NONE;
+	psa_key_derivation_operation_t *operation;
+
+	if (!checkContext(aContext, sizeof(psa_key_derivation_operation_t))) {
+		error = OT_ERROR_INVALID_ARGS;
+		goto exit;
+
+	}
+
+	operation = (psa_key_derivation_operation_t *) aContext->mContext;
+
+	*operation = psa_key_derivation_operation_init();
+
+	error = psaToOtError(psa_key_derivation_setup(operation, PSA_ALG_HKDF(PSA_ALG_SHA_256)));
+
+exit:
+	return error;
+}
+
+otError otPlatCryptoHkdfExtract(otCryptoContext   *aContext,
+				const uint8_t     *aSalt,
+				uint16_t           aSaltLength,
+				const otCryptoKey *aInputKey)
+{
+	otError error = OT_ERROR_NONE;
+	psa_status_t status = PSA_SUCCESS;
+	psa_key_derivation_operation_t *operation = NULL;
+	otCryptoKeyRef keyRef = PSA_KEY_ID_NULL;
```

Comment:
this file uses `snake_case` like most of Zephyr rather than `camelCase` (except for names that follow OpenThread)